behat.yml.dist

```diff
@@ -12,24 +12,24 @@ default:
                 - Tests\Integration\Bootstrap\KohanaContext
                 - Tests\Integration\Bootstrap\FeatureContext
                 - Tests\Integration\Bootstrap\RestContext:
-                    baseUrl: http://localhost:8000
+                    baseUrl: http://192.168.33.110
```

Comment:
Can you revert this? You shouldn't need to edit this file. You can copy it to `behat.yml` and edit that instead